vcs_remote_lookup_test.go

```diff
@@ -29,6 +29,7 @@ func TestVCSLookup(t *testing.T) {
 		"https://gopkg.in/tomb.v1":                                         {work: true, t: Git},
 		"https://golang.org/x/net":                                         {work: true, t: Git},
 		"https://speter.net/go/exp/math/dec/inf":                           {work: true, t: Git},
+		"git@github.com:Masterminds/vcs.git":                               {work: true, t: Git},
```

Comment:
could we also add a test with hostname other than github - for example `git@git.example.com:foo/bar.git`, to make sure that the Git VCS detection works correctly not only for github.com but also other git@&lt;hostname&gt; cases?  I would  really appreciate it.  (related to https://github.com/Masterminds/glide/issues/301)